website_portal_comment_attachment/static/src/js/portal_chatter.js

```diff
@@ -0,0 +1,65 @@
+odoo.define('website_portal_comment_attachment.chatter', function (require) {
+    'use strict';
+    var core = require('web.core');
+    var ajax = require('web.ajax');
+    var PortalChatter = require('portal.chatter').PortalChatter;
+
```

Comment:
```suggestion
    var _t = core._t;
```

Reply:
Is using directly core._t("Remove attachement") ok ?
If no, shall the same thing be done with `core.qweb`  ?